docs: pthon coprocessor user guide (#219)
GreptimeTeam/docs: docs/user-guide/python-coprocessor/query-data.md

# Query data

We provide two way for you to easily query data:

> 1. through plain SQL by write `query.sql(..)` :this returns a list[list [ PyVector ] ] that's convert from query result.

Reviewer: ```suggestion
1. through plain SQL by writing `query.sql(..)` :this returns a `list[list [ PyVector ] ]` which represents the converted query results.
```
